mackup/applications/music.cfg

```diff
@@ -0,0 +1,7 @@
+[application]
+name = Music
+
+[configuration_files]
+Library/Music/Scripts
```

Comment:
Apple software have an history of not behaving well with Mackup. What's in this folder?

Reply:
OSA scripts just like iTunes. It is true that from 10.4.0 – 10.4.3 the symlink caused iTunes scripts to [not work](https://github.com/lra/mackup/pull/1311), it has since been [resolved](https://github.com/lra/mackup/pull/1349) though.

Reply:
👌